apis/logging/v1/clusterlogging_types.go

```diff
@@ -124,13 +136,6 @@ type KibanaSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Kibana Resource Requirements",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:resourceRequirements"}
 	Resources *v1.ResourceRequirements `json:"resources"`
 
-	// Define which Nodes the Pods are scheduled on.
-	//
-	// +nullable
-	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Kibana Node Selector",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:nodeSelector"}
-	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
```

Comment:
These cannot be removed since it would result in a regression upon upgrade.  We have to:
* mark them as deprecated
* allow them to take precedence over those at the visualization level

The easiest solution for usage is to introduce a migration to migrate these fields into the new fields and then alter the remaining code to rely on the new fields... similar to what we did with the collector

Reply:
Done